gcc/config/riscv/t-arcv

```diff
@@ -0,0 +1,30 @@
+#                     rmx100mini  plain       rmx100_base.tcf - m                          rmx100_base.tcf                               rmx100_base.tcf + a                            rhx100_base.tcf                           rhx100_base_fpu.tcf - -zfh                 rhx100_base.tcf - -zcd + d                     rpx                                  rpx + fpu
+MULTILIB_OPTIONS    = march=rv32e/march=rv32i/march=rv32ic_zcb_zcmp_zcmt_zba_zbb_zbs_zicsr/march=rv32imc_zcb_zcmp_zcmt_zba_zbb_zbs_zicsr/march=rv32imac_zcb_zcmp_zcmt_zba_zbb_zbs_zicsr/march=rv32imac_zcb_zcmp_zba_zbb_zbs_zicsr/march=rv32imafc_zcb_zcmp_zba_zbb_zbs_zicsr/march=rv32imafd_zca_zcb_zcmp_zba_zbb_zbs_zicsr/march=rv64imac_zcb_zba_zbb_zbs_zicsr/march=rv64imafdc_zcb_zba_zbb_zbs_zicsr
```

Comment:
Just wondering if there's a way to have one option per line to improve readability. The same question (but to a lesser extent) is applicable to `MULTILIB_DIRNAMES`.

Reply:
Yes, this file is included by Makefile script. I will make it more readable.